contracts/package.json

```diff
@@ -11,8 +11,12 @@
     "node:hardhat": "hardhat node",
     "deploy:hardhat": "hardhat deploy --network hardhat",
     "deploy:mumbai": "hardhat deploy --network mumbai",
+    "deploy:sepolia": "hardhat deploy --network sepolia",
+    "deploy:goerli": "hardhat deploy --network goerli",
     "compile": "hardhat compile",
-    "verify:mumbai": "npx hardhat run ./scripts/verify-polyscan.js --network mumbai"
+    "verify:mumbai": "npx hardhat run ./scripts/verify-polyscan.js --network mumbai",
```

Comment:
I had actually changed the file's name to `verify-contract.js` as I found verify.js ambiguous but had forgotten to change these lines. I'll update them.

Reply:
I think `verify` is fine too since its in the `contracts/scripts` folder but if you already changed it's fine. But also if that was the case why are both files still there? We don't need both `verify.js` and `verify-contract.js`, we still need to delete on of those files. That was my point

Reply:
That was a typo I think because we never had a verify.js file so no need to remove it. I updated the commands so that they all use `verify-contract.js` now.

Reply:
There is  verify.js file, you can see it right in this PR? It was added as part of one of my original commits in the sepolia migration too remember we did it on a screenshare? So I did hte same change for goerli. But yes, if you look at the files in your PR, you can see verify.js is also there so we should remove it.